app/src/main/java/app/accrescent/client/ui/AppDetailsScreen.kt

```diff
@@ -215,10 +229,7 @@ fun AppDetails(
                 val totalMb = "%.1f".format(downloadProgress.total.toFloat() / 1_000_000)
 
                 Text("$partMb MB / $totalMb MB", Modifier.padding(top = 16.dp))
-            } else {
-                Spacer(modifier = Modifier.size(96.dp))
-                Text("", Modifier.padding(top = 16.dp))
-            }
```

Comment:
The spacer is intentionally here for padding so that when an installation progress indicator starts, it doesn't jolt the other UI elements around.